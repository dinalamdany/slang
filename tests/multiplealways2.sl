main()
{
	int counter = 0;
	init{
			#13
			print("TERMINATE!");
			print(counter);
			Terminate;
	}

	always {
		#3
		print("--every 3");
	}

	always{
		#1
		counter++;
		print("currently time:");
		print(counter);
	}
}