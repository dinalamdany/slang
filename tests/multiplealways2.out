currently time:
1
currently time:
2
--every 3
currently time:
3
currently time:
4
currently time:
5
--every 3
currently time:
6
currently time:
7
currently time:
8
--every 3
currently time:
9
currently time:
10
currently time:
11
--every 3
currently time:
12
TERMINATE!
12
